packages/lab/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -0,0 +1,227 @@
+      />
+      <MultilineInput readOnly fullBorder defaultValue="Value" {...args} />
+      <MultilineInput
+        readOnly
+        fullBorder
+        variant="secondary"
+        defaultValue="Value"
+        {...args}
+      />
+    </FlowLayout>
+  );
+};
+
+export const EmptyReadonlyMarker: ComponentStory<typeof MultilineInput> = (
+  args
+) => {
+  return (
+    <FlowLayout>
+      <MultilineInput readOnly={true} {...args} />
+      <MultilineInput readOnly={true} emptyReadOnlyMarker="*" {...args} />
+    </FlowLayout>
+  );
+};
+
+export const Placeholder: ComponentStory<typeof MultilineInput> = (args) => {
+  return <MultilineInput placeholder={"Enter a value"} {...args} />;
+};
+
+export const TextAlign: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ maxWidth: "366px" }}>
+      <MultilineInput defaultValue={args.defaultValue ?? "Value"} {...args} />
+      <MultilineInput
+        textAlign="center"
+        defaultValue={args.defaultValue ?? "Value"}
+        {...args}
+      />
+      <MultilineInput
+        textAlign="right"
+        defaultValue={args.defaultValue ?? "Value"}
+        {...args}
+      />
+    </FlowLayout>
+  );
+};
+
+export const ValidationStates: ComponentStory<typeof MultilineInput> = (
+  args
+) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput validationStatus="error" defaultValue="Value" {...args} />
+      <MultilineInput
+        fullBorder
+        validationStatus="error"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        validationStatus="warning"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        fullBorder
+        validationStatus="warning"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        validationStatus="success"
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        fullBorder
+        validationStatus="success"
+        defaultValue="Value"
+        {...args}
+      />
+    </FlowLayout>
+  );
+};
+
+export const WithAdornments: ComponentStory<typeof MultilineInput> = (args) => {
+  return (
+    <FlowLayout style={{ width: "366px" }}>
+      <MultilineInput
+        startAdornment={<Button variant="cta">AB</Button>}
+        endAdornment={
+          <>
+            <Text>%</Text>
+            <FilterClearIcon />
+          </>
+        }
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        startAdornment={<FlagIcon />}
+        endAdornment={
+          <Button>
+            <BookmarkSolidIcon />
+          </Button>
+        }
+        defaultValue="Value"
+        {...args}
+      />
+      <MultilineInput
+        readOnly
+        startAdornment={<FlagIcon />}
+        endAdornment={
+          <Button disabled>
+            <BookmarkSolidIcon />
+          </Button>
+        }
+        defaultValue="Value"
```

Comment:
```suggestion
        defaultValue="ReadOnly Value"
```

Reply:
I think these will change with the guidance coming from content soon to some realistic examples